test/lsp.py

```diff
@@ -60,10 +60,7 @@ def receive_message(self) -> Union[None, dict]:
                 # server quit
                 return None
             line = line.decode("utf-8")
-            if not line.endswith("\r\n"):
```

Comment:
Why did you remove this check?

Reply:
the rstrip does also remove CRLF but more relaxed, such that it would have allowed a simple LF too. I've reverted it though.